assets/js/solcast.webflow.footer.js

```diff
@@ -0,0 +1,23 @@
+// Footer Newsletter Subscription Popup watcher to resize iFrame.
+
+const targetElement = document.querySelector('.footer_subscribe-popup');
+
+if (targetElement) {
+    // Create a new MutationObserver
+    const observer = new MutationObserver((mutations) => {
+        mutations.forEach((mutation) => {
+            if (mutation.type === 'attributes' && mutation.attributeName === 'style') {
+                const displayStyle = window.getComputedStyle(targetElement).display;
+                if (displayStyle === 'flex') {
+                    iFrameResize({log:true});
```

Comment:
The comments here could be more meaningful about WHY we need to resize the footer popup, but mostly I can't see where `iFrameResize` is defined? 

Should this be in the same file or if not, where should other developers look to find it?

Reply:
@alex-solcast I've added some additional comments to the js file. I believe these should be 2 separate js files as the iframe.resizer.js script was created by Salesforce.